apps/api/src/purchase-orders/purchase-orders.service.ts

```diff
@@ -0,0 +1,90 @@
+import { Injectable, NotFoundException, Logger } from '@nestjs/common';
+import { PurchaseOrders } from "@prisma/client";
+import { CreatePurchaseOrderDto } from './dto/create-purchase-order.dto';
+import { UpdatePurchaseOrderDto } from './dto/update-purchase-order.dto';
+import { PrismaService } from "../prisma.service";
+
+@Injectable()
+export class PurchaseOrdersService {
+  constructor(private prisma: PrismaService) {
+  }
+
+  // Just a note that the naming convention here seems inconsistent. In prisma, PurchaseOrders is plural while ParentItem is singular
+  async findAll(sortField: string, sortOrder: string): Promise<PurchaseOrders[]> {
+
+    const purchaseOrders = await this.prisma.purchaseOrders.findMany({
+      /* 
```

Comment:
Wanted to highlight this comment and ensure I'm not missing something.

Reply:
Updating the seed data to store a unix timestamp would resolve the issue. Not ideal since it makes the dates not readable if you query the records directly. The real issue will be solved when prisma updates their code to support DateTime -> Text mapping